pkg/agent/route/route.go

```diff
@@ -0,0 +1,293 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package route
+
+import (
+	"fmt"
+	"golang.org/x/sys/unix"
+	"net"
+	"os"
+	"strings"
+
+	"k8s.io/klog"
+
+	"github.com/vishvananda/netlink"
+	"github.com/vmware-tanzu/antrea/pkg/agent/iptables"
+	"github.com/vmware-tanzu/antrea/pkg/agent/types"
+	"github.com/vmware-tanzu/antrea/pkg/agent/util"
+)
+
+const (
+	// AntreaServiceTable is route table name for Antrea service traffic.
+	AntreaServiceTable = "Antrea-service"
+	// AntreaServiceTableIdx is route table index for Antrea service traffic.
+	AntreaServiceTableIdx = 300
+)
+
+// Client is route client.
+type Client struct {
+	nodeConfig *types.NodeConfig
+}
+
+type serviceRtTableConfig struct {
+	Idx  int
+	Name string
+}
+
+func (s *serviceRtTableConfig) String() string {
+	return fmt.Sprintf("%s: idx %d", s.Name, s.Idx)
+}
+
+func (s *serviceRtTableConfig) IsMainTable() bool {
+	return s.Name == "main"
+}
+
+var (
+	// ServiceRtTable contains Antrea service route table information.
+	ServiceRtTable = &serviceRtTableConfig{Idx: 254, Name: "main"}
```

Comment:
Is ServiceRtTable used outside the route package? If not make it internal - serviceRtTable.

Reply:
ServiceRtTable is also used in integration test. Otherwise it can be made local